collect_app/src/main/java/org/odk/collect/android/database/DatabaseConstants.java

```diff
@@ -3,7 +3,7 @@
 public class DatabaseConstants {
 
     public static final String FORMS_TABLE_NAME = "forms";
-    public static final int FORMS_DATABASE_VERSION = 9;
+    public static final int FORMS_DATABASE_VERSION = 10;
```

Comment:
My thinking is we need a migration here as we already have version 9 of the database out in the wild.

Reply:
Yes, absolutely.